src/app/reducers.ts

```diff
@@ -7,6 +7,8 @@ import { REMOTE_CONFIG_VALUES_DEFAULTS } from 'src/firebase/remoteConfigValuesDe
 import { Screens } from 'src/navigator/Screens'
 import { getRehydratePayload, REHYDRATE, RehydrateAction } from 'src/redux/persist-helper'
 
+const PERSISTED_DEEP_LINKS = ['https://valoraapp.com/share', 'celo://wallet/jumpstart']
+
```

Comment:
I think we don't care so much about the scheme and domain, it's more the `/path` part

Reply:
is it harmful to have the whole url check? i wanted to avoid complexity here, the `handleDeepLink` saga has some custom logic for parsing the urls which i wanted to avoid here (it uses a deprecated `parse` method to parse the deep link)...

Reply:
would that enable any link with `/share` pathname to be persisted? e.g. `https://evil-resource.net/share`

Reply:
Valora is only registered to handle `celo://` or specific `https://valoraapp.com` links.
So it won't ever handle `https://evil-resource.net/share`.

I guess we can always add a dynamic config, but would have been nice without it.

Reply:
do you feel strongly that we should avoid checking the domain @jeanregisser ? if so, without doing the dynamic config, i think it makes sense to do the same check as in the deep link handler - in which case we'd need to extract the logic.

what this gives us is the ability to handle `celo://wallet/share` and `https://valoraapp.com/jumpstart`, both of which i'm not sure we have immediate plans to support. it's not a lot of work to do so i don't mind doing it, but i guess i'm not totally convinced of the value

Reply:
although writing it now, i think we should use a dynamic link for the jumpstart links rather than the `celo://` link :|

Reply:
i'll go ahead and merge this, and update the links once we know more